tests/cnf/core/network/security/tests/nftables-custom-firewall-test.go

```diff
@@ -374,49 +453,52 @@ func addDeleteStaticRouteOnWorkerNodes(testPodList []*pod.Builder, routeMap map[
 	}
 }
 
-func activateNftablesIfInactive() {
+func activateNftablesIfInactive(targetNodeName string) {
 	// Get the current status of the nftables service
-	statuses, err := cluster.ExecCmdWithStdoutWithRetries(APIClient,
-		3, 10*time.Second,
-		"systemctl is-active nftables.service | cat -")
+	statuses, err := cluster.ExecCmdWithStdout(APIClient,
+		"systemctl is-active nftables.service | cat -",
+		metav1.ListOptions{LabelSelector: "kubernetes.io/hostname=worker-0"})
```

Comment:
use targetNodeName instead of worker-0

Reply:
added KubernetesHostNameLableWork0 to tsparams